notebooks/09_Monitoring/src/09_02_Iris_Pipeline_Metrics/component.yaml

```diff
@@ -0,0 +1,15 @@
+name: 'IRIS Classification Pipeline Metrics Task'
```

Comment:
Can we use containerOP instead? As I understand, we just need a code snippet to showcase this feature. Try not to bring to many components. 

If there's breaking change in KFP manifest, we have to migrate these components.

Reply:
Fixed in the latest version of PR.